dashboard/app/models/user.rb

```diff
@@ -246,6 +247,19 @@ def delete_course_as_facilitator(course)
     courses_as_facilitator.find_by(course: course).try(:destroy)
   end
 
+  def log_admin_save
+    # Do not log for development or adhoc environments.
+    return unless [:test, :staging, :levelbuilder, :production].include? rack_env
```

Comment:
This should not include `:test`, otherwise we'll be spammed with messages from automated testing.